package.json

```diff
@@ -8,6 +8,9 @@
     "yarn": ">=1.15.*"
   },
   "dependencies": {
+    "@ant-design/icons": "^4.2.2",
+    "@material-ui/core": "^4.11.0",
```

Comment:
Hahaha gotcha! 
Might want to remove these
"yarn remove(or uninstall) @material-ui/core" and the package below this on line 13

Reply:
Guilty as charged, these are now removed 👍